.meta/dietpi-imager

```diff
@@ -261,6 +261,35 @@ Do you want to overwrite or backup the existing file to /root/$OUTPUT_IMG_NAME.$
 		IMAGE_SIZE=$(( ( $(sfdisk -qlo End $FP_SOURCE | tail -1) + 1 ) * 512 )) # 512 byte sectors => Byte
 		IMAGE_SIZE=$(( $IMAGE_SIZE + ( 512 * 256 ) )) # 64 byte for secondary GPT + safety net
 
+		# Check if there is enough free disk space to store the image and the 7z file
+		# Image file source
+		if [[ $FP_SOURCE_IMG ]]; then
+
+			NEEDED_FREE_SPACE=$(( $IMAGE_SIZE / 100 * 15 ))
+
+		# Drive source, clone with dd
+		elif [[ $CLONING_TOOL == 'dd' ]]; then
+
+			NEEDED_FREE_SPACE=$(( $IMAGE_SIZE / 100 * 115 ))
+
+		# Drive source, clone with Clonezilla
+		else
+
+			NEEDED_FREE_SPACE=$(( $IMAGE_SIZE / 100 * 23 * 2 + 330000000 * 3 ))
+
+		fi
+		NEEDED_FREE_SPACE=$(( $NEEDED_FREE_SPACE / 1024 / 1024 ))
+		CURRENT_FREE_SPACE=$(( $(df --output=avail /root | tail -n1 | mawk '{print $1}') / 1024 ))
```

Comment:
@sal666 
Lets use `G_CHECK_FREESPACE`: https://github.com/MichaIng/DietPi/blob/dev/dietpi/func/dietpi-globals#L1807
```
G_CHECK_FREESPACE /root $NEEDED_FREE_SPACE || Exit_On_Fail
```
```
2019-11-21 21:43:02 root@micha:/var/lib# G_CHECK_FREESPACE /root 1000
[  OK  ] Free space check: path=/root | available=58753 MiB | required=1000 MiB
2019-11-21 21:43:16 root@micha:/var/lib# G_CHECK_FREESPACE /root 100000
[FAILED] Free space check: path=/root | available=58753 MiB | required=100000 MiB
```

The method is is exactly the same, output same information 😃.

Reply:
Nice +1